Makefile

```diff
@@ -220,7 +220,7 @@ ifeq (, $(shell which controller-gen))
 	CONTROLLER_GEN_TMP_DIR=$$(mktemp -d) ;\
 	cd $$CONTROLLER_GEN_TMP_DIR ;\
 	go mod init tmp ;\
-	go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.3.0 ;\
+	go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.9.0 ;\
```

Comment:
```suggestion
	go install sigs.k8s.io/controller-tools/cmd/controller-gen@v0.9.0 ;\
```

Reply:
btw, I don't think Makefile should install tools. I'd leave it just checking if tool is available. 
Now it only installs it, but doesn't check if already existing one is in correct version.
Update is required to be done manually anyway.